crates/net/src/sim2h_worker.rs

```diff
@@ -82,7 +82,7 @@ impl Sim2hWorker {
         agent_id: Address,
         conductor_api: ConductorApi,
     ) -> NetResult<Self> {
-        let reconnect_interval = Duration::from_secs(INITIAL_CONNECTION_TIMEOUT_MS);
+        let reconnect_interval = Duration::from_millis(INITIAL_CONNECTION_TIMEOUT_MS);
```

Comment:
Ouch that's embarrassing we all missed that!

Reply:
yep.